### ISSUE TITLE
Results on PhotoTour

### ISSUE THREAD

**Issue by mfaisal59** [xuyanwu/RAL-Net#2]
Hi,

Since you have not provided your pre-trained models, so we re-trained all the models again using the following code:

python RAL_Net.py --data-root='your data root' --epochs=10 --batch-size=512 --n-pairs=5000000 --loss-type=RAL_loss --lr=10 --augmentation=True

The results we obtained are different than the one reported in the paper (shown in brakets "( )" ).

**Training on Liberty:**
FPR95 on Notredame = 0.414 --(0.37)
FPR95 on Yosmite = 1.5879 -- (1.52)

**Training on Notredame :** 
FPR95 on Liberty= 1.248 -- (1.30)
FPR95 on Yosmite = 1.336 -- (1.31)

**Training on Yosmite :**
FPR95 on Notredame = 0.706 -- (0.67)
FPR95 on Liberty = 2.558 -- (2.39)

Can you please provide the pre-trained models.
Also, which model do you use to evaluate on HPatches dataset.

Thanks in Advance.

**Comment by xuyanwu** [comment]
The results are within the normal range (differences to our results are around 0.05). Since the algorithms are trained using SGD, the results cannot be the same as those reported in the paper. 

We used the model trained from Liberty data, and the details are in the paper.

**Comment by mfaisal59** [comment]
Can you please provide the pre-trained models.

**Comment by xuyanwu** [comment]
Could you please send me your email? I will send you my pre-trained liberty model.

**Comment by xuyanwu** [comment]
[RAL_loss_9_5000000_512_liberty.th.zip](https://github.com/xuyanwu/RAL-Net/files/3049146/RAL_loss_9_5000000_512_liberty.th.zip)

**Comment by mfaisal59** [comment]
Thank you.
